packages/kernels/linux-rpi/build.yaml

```diff
@@ -0,0 +1,17 @@
+package_dir: /out
+
+image: ubuntu:24.04
+
+prelude:
+  - apt-get update -y && apt-get install -y gcc gcc-aarch64-linux-gnu git make flex bc bison libssl-dev xz-utils kmod
+  - git clone --branch {{ .Values.version }} --depth 1 https://github.com/raspberrypi/linux.git linux
```

Comment:
This pulls their default branch (`rpi-6.6.y` ?). Should we make it a value to avoid surprises? (so that we can manually bump)

Reply:
I'm stupid. You already do :facepalm: